recipes/computer_vision/object_detection/model_server/object_detection_server.py

```diff
@@ -1,19 +1,30 @@
-from transformers import AutoImageProcessor, AutoModelForObjectDetection
+from transformers import AutoImageProcessor, AutoModelForObjectDetection, AutoConfig
```

Comment:
I don't think autoConfig is used. This can be removed.

Reply:
Will do